genet/validate/network.py

```diff
@@ -35,3 +36,33 @@ def describe_graph_connectivity(G):
     # find number of connected subgraphs
     dict_to_return['number_of_connected_subgraphs'] = len(find_connected_subgraphs(G))
     return dict_to_return
+
+
+def zero_value(value):
+    return value in {0, '0', '0.0'}
```

Comment:
I guess this fails for the string `"0.00"`? If that might turn out to be important, you could parse the string to an int and then check for equality to 0.

Reply:
mm, good point